server/src/assets/assets.controller.ts

```diff
@@ -34,9 +36,12 @@ export class AssetsController {
   }
 
   @Get()
-  findAll(): Promise<Asset[]> {
-    return this.assetsService.findAll();
+  get(@Query() getAssetsDto: GetAssetsDto): Promise<Asset[]> {
+    return this.assetsService.getAssets(getAssetsDto);
   }
+  // findAll(): Promise<Asset[]> {
+  //   return this.assetsService.findAll();
+  // }
```

Comment:
remove

Reply:
oops! Thanks! 
Do we still need the findAll() method in assets.service.ts if it's not used here? Is there any downside to leaving it there in case users do want to see all assets?

Reply:
we should remove any unused code. less future wasted work maintaining something that's not being used